include/Smilies.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+/**
+ * @file include/Smilies.php
+ * @brief This file contains the Smilies class which contains functions to handle smiles
+ */
+
+/**
+ * This class contains functions to handle smiles
```

Comment:
There is at least my emojione addon, but as it uses the hook as well it should work as well as the others